### PR TITLE
Support extra settings in service/bin

### LINKED ISSUE #42: Generate and source service/bin/config.sh as part of init.sh execution
- Generate service/bin/config.sh as part of script generation.
- Add `javaHome` config option and emit `JAVA_HOME="<javaHome>"` into config.sh as part of generation

### PR DESCRIPTION
**Pull request by markelliot**
fixes #42

### CODE REVIEW

File: src/main/resources/config.sh [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# override the JAVA_HOME value with a known JRE location
+@javaHome@
+
+# use JVM_OPTS to set JVM settings, such as minimum and maximum heap
+#JVM_OPTS="-Xms1g -Xmx1g"
```
Comment (bavardage) [inline src/main/resources/config.sh:5]: wait but how do we actually set the JVM_OPTS?

Comment (markelliot) [inline src/main/resources/config.sh:5]: The idea is that this file is easy to edit. If you want to set default JVM opts can do it through the normal config in gradle.

Comment (bavardage) [inline src/main/resources/config.sh:5]: aha yes - may well not be deploying in a managed way - could just be extracting the generated tgz

**Comment by bavardage** [comment]
:+1:

**Pull request merged**